libs/utils/platforms/hikey960.json

```diff
@@ -0,0 +1,37 @@
+{
+	// HiKey960 boards have two clusters
+	// 4 Cortex A73 + 4 Cortex A53 big.LITTLE architecture
+	"board" : {
+		"cores" : [
+			"A53_0", "A53_0", "A53_0", "A53_0",
+			"A73_1", "A73_1", "A73_1", "A73_1"
+		],
+		"big_core" : "A73_1",
+		"modules"  : [ "bl", "cpufreq", "cpuidle" ]
+	},
+
+    "nrg_model" : {
+        "little" : {
+            "cluster" : {
+                "nrg_max" :   70,
+            },
+	        "cpu" : {
+                "nrg_max" :   1210,
```

Comment:
Don't know why you point to hi3660-sched-energy.dtsi, which is CA72 + CA53 CPUs.
However, Hikey960 is CA73+CA53. 
http://www.96boards.org/product/hikey960/

Even if the energy model can be tuned later, we can check in the support for hikey960 support first of all.